docs/resources.js

```diff
@@ -24,58 +15,11 @@ function getDisaster() {
 
 /** @return {string} EE asset path for score asset for the current disaster */
 function getScoreAsset() {
-  return gdEePathPrefix + getDisaster() + '/data-ms-as-nod';
+  // TODO(janakr): Modify map to handle no-damage scenario more intuitively.
+  //  Should make damage threshold, weight 0, probably hide those toggles
+  //  completely. Other changes?
+  return gdEePathPrefix + getDisaster() + '/data-ms-as-tot';
```

Comment:
Maybe make this path a constant so it's not repeated in resources.js?

Reply:
Did this change?

Reply:
Yes, import_data is now just calling this to get the correct path.